apps/web-connect/src/features/checkout/components/ChooseQuantityRow.tsx

```diff
@@ -16,6 +16,14 @@ export function ChooseQuantityRow(): JSX.Element {
     // Destructure necessary values from the WebBuyKeysContext
     const { quantity, setQuantity, maxSupply } = useWebBuyKeysContext();
 
+    function calculateMaxPurchase():number {
+        if(maxSupply >= 175) {
+            return 175;
+        }else{
+            return maxSupply;
```

Comment:
Please make this a constant in this file at least so we just have to change it at one place in the future